NestedLoops3/07.SafePasswordsGenerator/07.SafePasswordsGenerator.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace _07.SafePasswordsGenerator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int a = int.Parse(Console.ReadLine());
+            int b = int.Parse(Console.ReadLine());
+            int maxNumberOfGeneratedPasswords = int.Parse(Console.ReadLine());
+
+            char firstSymbol = '#';
+            char secondSymbol = '@';
+            int counter = 0;
+
+            for (int firstNumber = 1; firstNumber <= a; firstNumber++)
+            {
+                for (int secondNumber = 1; secondNumber <= b; secondNumber++)
+                {
+                    counter++;
+                    Console.Write($"{firstSymbol}{secondSymbol}{firstNumber}{secondNumber}{secondSymbol}{firstSymbol}|");
+
+                    if (maxNumberOfGeneratedPasswords == counter)
+                    {
+                        return;
```

Comment:
If you want to stop the program, use Environment.Exit(0), if you just want to exit two for loops, use bool variable in outside loop and set it to false and break in inside loop